daemons/gptp/common/ptp_message.cpp

```diff
@@ -1611,23 +1616,31 @@ void PTPMessagePathDelayRespFollowUp::processMessage(IEEE1588Port * port)
 		Timestamp prev_peer_ts_theirs;
 		FrequencyRatio rate_offset;
 		if( port->getPeerOffset( prev_peer_ts_mine, prev_peer_ts_theirs )) {
+			FrequencyRatio upper_ratio_limit, lower_ratio_limit;
+			upper_ratio_limit = PPM_OFFSET_TO_RATIO(UPPER_LIMIT_PPM);
+			lower_ratio_limit = PPM_OFFSET_TO_RATIO(LOWER_LIMIT_PPM);
+
 			mine_elapsed =  TIMESTAMP_TO_NS(request_tx_timestamp)-TIMESTAMP_TO_NS(prev_peer_ts_mine);
 			theirs_elapsed = TIMESTAMP_TO_NS(remote_req_rx_timestamp)-TIMESTAMP_TO_NS(prev_peer_ts_theirs);
 			theirs_elapsed -= port->getLinkDelay();
 			theirs_elapsed += link_delay < 0 ? 0 : link_delay;
 			rate_offset =  ((FrequencyRatio) mine_elapsed)/theirs_elapsed;
-			port->setPeerRateOffset(rate_offset);
-			if (!port->getAutomotiveProfile()) {
-				if( !port->setLinkDelay( link_delay ) ) {
-					GPTP_LOG_ERROR("Link delay %ld beyond neighborPropDelayThresh; not AsCapable", link_delay);
-					port->setAsCapable( false );
-				} else {
-					port->setAsCapable( true );
-				}
+
+			if( rate_offset < upper_ratio_limit && rate_offset > lower_ratio_limit ) {
+				port->setPeerRateOffset(rate_offset);
 			}
 		}
 	}
-
+	if( !port->setLinkDelay( link_delay ) ) {
```

Comment:
I noticed this when I was troubleshooting the clock frequency adjustment issue so I can't say the level of impact of this but the link delay is used in calculations so yes it would lead to some error in calculations.